### PR TITLE
aireplay-ng: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/aireplay-ng.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# aireplay-ng
+
+> Inyecta paquetes en una red inalámbrica.
+> Parte de `aircrack-ng`.
+> Más información: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
+
+- Envía una cantidad específica de paquetes disociados dada la dirección MAC de un punto de acceso, la dirección MAC de un cliente y una interfaz:
+
+`sudo aireplay-ng --deauth {{count}} --bssid {{ap_mac}} --dmac {{client_mac}} {{interface}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/aireplay-ng.md:9]:
```suggestion
`sudo aireplay-ng --deauth {{1..infinity}} --bssid {{ap_mac}} --dmac {{client_mac}} {{interface}}`
```

Optional fix: [range syntax](https://github.com/tldr-pages/tldr/pull/9646) can be used.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🚀

File: pages.es/common/aireplay-ng.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# aireplay-ng
+
+> Inyecta paquetes en una red inalámbrica.
+> Parte de `aircrack-ng`.
+> Más información: <https://www.aircrack-ng.org/doku.php?id=aireplay-ng>.
+
+- Envía una cantidad específica de paquetes disociados dada la dirección MAC de un punto de acceso, la dirección MAC de un cliente y una interfaz:
+
+`sudo aireplay-ng --deauth {{1..infinity}} --bssid {{ap_mac}} --dmac {{client_mac}} {{interface}}`
```
Comment (navarroaxel) [inline pages.es/common/aireplay-ng.md:9]: `infinity` is not a specific quantity, `infinity` should be another example in the English page and here.
```suggestion
`sudo aireplay-ng --deauth {{cantidad}} --bssid {{mac_punto_acceso}} --dmac {{mac_cliente}} {{interfaz}}`
```
Translate placeholders.

**Comment by EmilySeville7cfg** [comment]
#9702 is created to use range syntax. @kant, if u wanna to keep translation synced you can update the translation page again. 😄

**Review by navarroaxel — APPROVED** [review]

**Comment by kant** [comment]
> #9702 is created to use range syntax. @kant, if u wanna to keep translation synced you can update the translation page again. 😄

I will :)

**Comment by EmilySeville7cfg** [comment]
@kant, range syntax has been deprecated as some contributors (@sbrl) though that it's too complex. Just to notice.

**Pull request merged**